[2023-04-07 13:06:00,721] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:06:00] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:06:00,788] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:06:00] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:06:00,847] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:06:00] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:06:05,617] ERROR log_exception() L1449 Exception on /servico [PUT] - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py", line 1520, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py", line 1539, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py", line 1695, in make_response
    raise TypeError(
TypeError: The view function for 'put_servico' did not return a valid response. The function either returned None or ended without a return statement.
[2023-04-07 13:06:05,619] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:06:05] "[35m[1mPUT /servico HTTP/1.1[0m" 500 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:06:36,020] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:06:36] "GET /servicos HTTP/1.1" 200 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:06:43,759] ERROR log_exception() L1449 Exception on /servico [PUT] - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py", line 1520, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py", line 1539, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\flask\app.py", line 1695, in make_response
    raise TypeError(
TypeError: The view function for 'put_servico' did not return a valid response. The function either returned None or ended without a return statement.
[2023-04-07 13:06:43,760] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:06:43] "[35m[1mPUT /servico HTTP/1.1[0m" 500 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:07:50,890] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.15.138:5000 (Press CTRL+C to quit) - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:07:53,968] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:07:53] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:07:54,093] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:07:54] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:07:54,102] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:07:54] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:07:54,103] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:07:54] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:07:54,168] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:07:54] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:07:54,224] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:07:54] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:08:03,629] WARNING put_servico() L388  Erro ao editar o serviço 'Escova', Serviço já cadastrado na base - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\app.py L388 
[2023-04-07 13:08:03,629] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:08:03] "[31m[1mPUT /servico HTTP/1.1[0m" 400 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:08:18,087] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:08:18] "GET /servicos HTTP/1.1" 200 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:08:23,924] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:08:23] "[35m[1mPUT /servico HTTP/1.1[0m" 204 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:08:32,823] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:08:32] "GET /servicos HTTP/1.1" 200 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:08:41,432] WARNING get_servico() L454  Erro ao buscar o servico 'pintar a unha', Serviço não encontrado na base :/ - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\app.py L454 
[2023-04-07 13:08:41,433] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:08:41] "[33mGET /servico?descricao=pintar%20a%20unha HTTP/1.1[0m" 404 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:08:48,605] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:08:48] "GET /servico?descricao=Escovas HTTP/1.1" 200 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:12:26,272] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.15.138:5000 (Press CTRL+C to quit) - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:12:30,038] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:12:30] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:12:30,134] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:12:30] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:12:30,140] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:12:30] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:12:30,141] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:12:30] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:12:30,207] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:12:30] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-07 13:12:30,265] INFO _log() L224  127.0.0.1 - - [07/Apr/2023 13:12:30] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\PUC-RIO\Sprint-1\BackEnd\puc-rio-mvp-sprint-1-back-end\venv\Lib\site-packages\werkzeug\_internal.py L224 
